packages/node-cli/src/validation/InteractiveAnswersValidator.js

```diff
@@ -133,7 +133,7 @@ class InteractiveAnswersValidator {
 	}
 
 	validateAuthIDNotInList(newAuthID, authIDs) {
-		return !authIDs.includes(newAuthID)
+		return !authIDs.hasOwnProperty(newAuthID)
```

Comment:
This is used in ManagaAccountCommnandGenerator.js line 152 : `(fieldValue) => validateAuthIDNotInList(fieldValue, Object.keys(authIDList))`
Object.keys returns an array with the names of the authIDList, so when checking "ownproperties" of an array your are not actually cheking for the values (names of the authids) but for its indexes ( 0, 1, ... n) . 
The easy check is to run `suitecloud account:manageauth -i` and try to rename an authid to 0 or 1.

Reply:
We will need to use the same approach to use this validator from manageauth and setup.  So either pas a map of the authids or an array with their names from both placess.

Reply:
Resolved... I change some variable names from List to Map, because I usually think in List as an Array.